.github/workflows/pre-submit.e2e.generic.workflow.yml

```diff
@@ -14,7 +14,8 @@ jobs:
       actions: read # For reading workflow info.
     uses: ./.github/workflows/slsa2_provenance.yml
     with:
-      subjects: "2e0390eb024a52963db7b95e84a9c2b12c004054a7bad9a97ec0c7c89d4681d2    binary-name"
+      # echo "2e0390eb024a52963db7b95e84a9c2b12c004054a7bad9a97ec0c7c89d4681d2    binary-name" | base64 -w0
+      subjects: "MmUwMzkwZWIwMjRhNTI5NjNkYjdiOTVlODRhOWMyYjEyYzAwNDA1NGE3YmFkOWE5N2VjMGM3Yzg5ZDQ2ODFkMiAgICBiaW5hcnktbmFtZQo="
```

Comment:
wondering if this should be labeled as "base64-subjects"? or something indicated the encoding in the name?

Reply:
or have this part of the documentation of inputs?

Reply:
Why not both?

Done.